scripts/run_qwen3_4b_fsdp.py

```diff
@@ -18,7 +18,7 @@ class ScriptArgs:
     num_gpus_per_node: int = 8
     hardware: Literal["H100"] = "H100"
     extra_args: str = ""
-    multi_eval: bool = True
+    multi_eval: bool = False
```

Comment:
for simplicity i make multi eval non-default. but that does not harm and is completely unrelated (just to save a bit of time for users who use default flag), since it is just evaling one or multi benchmark.